modules/resource_group/resource_group.tf

```diff
@@ -2,3 +2,43 @@ resource "azurerm_resource_group" "demo" {
   name     = "${var.name}"
   location = "${var.location}"
 }
+
+resource "azurerm_network_security_group" "demo-west" {
+  name                = "consul-west"
+  location            = "West US"
+  resource_group_name = "${azurerm_resource_group.demo.name}"
+}
```

Comment:
Should this be refactored to include all 3 datacenters and maybe be moved to the networking module?